src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/QualityGuidelines/DoNotInitializeUnnecessarily.cs

```diff
@@ -58,6 +59,7 @@ public sealed override void Initialize(AnalysisContext context)
                 IPropertySymbol? prop = init.InitializedProperties.FirstOrDefault();
                 if (prop != null &&
                     init.Value != null &&
+                    prop.ContainingType?.TypeKind != TypeKind.Struct && // Do not report on instance members of struct
```

Comment:
The comment mentions *instance* members, but the check is for structs in general, including static and instance members. Anything I'm missing? @stephentoub @Evangelink

Reply:
Ideally the rule would still flag statics.

Reply:
Damn, sorry I missed that. Will look into it tonight (CET time).

Reply:
It's now fixed.